backend/src/main/java/gov/cdc/usds/simplereport/service/model/reportstream/FeedbackMessage.java

```diff
@@ -2,28 +2,30 @@
 
 import java.io.Serializable;
 import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.ToString;
 
 @Getter
 @Setter
 @ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
 public class FeedbackMessage implements Serializable {
+  public enum ErrorType {
+    MISSING_HEADER,
+    MISSING_DATA,
+    INVALID_DATA
```

Comment:
Awesome, I'm glad you were able to identify these broad categories from the [content doc](https://docs.google.com/spreadsheets/d/12ZgrNl7Cdnxd-T1nO7XDuiSVVLKH7sOkfGYJ_wYtjro/edit#gid=2012187161). This will make classifying these errors much simpler.